Description: Upstream changes introduced in version 0.0.1+svn20110409.0930-7yavdr1~preciseubuntu1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 eventlircd (0.0.1+svn20110409.0930-7yavdr1~preciseubuntu1) precise; urgency=medium
 .
   * changed path for lircd2uinput
 .
 The person named in the Author field signed this changelog entry.
Author: Alexander Grothe <alexander.grothe@gmx.net>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- eventlircd-0.0.1+svn20110409.0930.orig/udev/lircd_helper.in
+++ eventlircd-0.0.1+svn20110409.0930/udev/lircd_helper.in
@@ -47,7 +47,7 @@ case "${ACTION}" in
                 devlink_instance=`echo ${devlink} | /bin/sed -e 's/\/\+/~/g' -e 's/^~dev~//'`
                 rm -f "@localstatedir@/run/lirc/lircd-${devlink_instance}"
                 ln -s "lircd-${devname_instance}" "@localstatedir@/run/lirc/lircd-${devlink_instance}"
-            /usr/bin/lircd2uinput.py --lircd-socket="/var/run/lirc/lircd-${devname_instance}" &
+            /usr/bin/lircd2uinput --lircd-socket="/var/run/lirc/lircd-${devname_instance}" &
             done
         fi
         ;;
